Source/WebKit/GPUProcess/GPUConnectionToWebProcess.cpp

```diff
@@ -647,8 +647,14 @@ void GPUConnectionToWebProcess::createRemoteGPU(WebGPUIdentifier identifier, Ren
     auto* renderingBackend = it->value.get();
 
     auto addResult = m_remoteGPUMap.ensure(identifier, [&] {
-        return IPC::ScopedActiveMessageReceiveQueue { RemoteGPU::create([this] (MediaPlayerIdentifier identifier, Function<void(MediaPlayer&)>&& callback) mutable {
-            this->performWithMediaPlayerOnMainThread(identifier, WTFMove(callback));
+        return IPC::ScopedActiveMessageReceiveQueue { RemoteGPU::create([this] (std::variant<WebCore::MediaPlayerIdentifier, WebKit::RemoteVideoFrameReference> identifier, Function<void(RefPtr<WebCore::VideoFrame>)>&& callback) mutable {
+            return WTF::switchOn(identifier, [&callback, this] (WebCore::MediaPlayerIdentifier i) {
```

Comment:
you're already capture by reference the line above, may as well continue, it's shorter/neater and will capture the same thing